### ISSUE TITLE
Add TPC-E support

### ISSUE THREAD

**Issue by bpkroth** [cmu-db/benchbase#177]
Would be nice to add TPC-E as a supported benchmark as well.

**Comment by apavlo** [comment]
I totally agree with you. We have an incomplete implementation in Java for H-Store but it is over 10 years old:

https://github.com/apavlo/h-store/tree/master/src/benchmarks/edu/brown/benchmark/tpce/

Nothing comes up on Google for another Java implementation.

**Comment by bryanosborne** [comment]
I second this request!

**Comment by apavlo** [comment]
WIP TPC-E Branch from @xinhao_dev: https://github.com/xinhaoc/benchbase/tree/xinhao_dev
